binary_wheel_builder/api/build.py

```diff
@@ -22,7 +24,7 @@ def _write_wheel(
         metadata: dict,
         wheel_file_entries: list[WheelFileEntry]
 ):
-    wheel_file_path = os.path.join(out_dir, wheel.wheel_filename(tag))
+    wheel_file_path = Path(out_dir) / wheel.wheel_filename(tag))
```

Comment:
chore: Please add the import for `pathlib.Path` 

Like the object-oriented Path api more, don't recall why I did not use it here, but dont see a reason not to.